tests/foreman/cli/test_activationkey.py

```diff
@@ -74,7 +74,9 @@ def test_positive_create_with_description(desc, module_org, module_target_sat):
 
 
 @pytest.mark.tier1
-def test_positive_create_with_default_lce_by_id(module_org, get_default_env, target_sat):
+def test_positive_create_with_default_lce_by_id(
+    module_org, get_default_env, target_sat, module_promoted_cv
```

Comment:
I'd suggest keeping the fixture scope same, it can cause issue like different satellite being used during test 
 execution. Also, `get_default_env ` and `module_promoted_cv` uses `module_target_sat`.
```suggestion
    module_org, get_default_env, module_target_sat, module_promoted_cv
```